src/main/java/com/svydovets/bibirnate/session/Session.java

```diff
@@ -0,0 +1,5 @@
+package com.svydovets.bibirnate.session;
+
+public interface Session {
+    <T> T findById(Class<T> type, Object id);
```

Comment:
What do you think about swapping params? First we pass object and after that its class